spec/index.bs

```diff
@@ -4913,6 +4913,13 @@ same module, they should avoid passing that information to
             **Returns:** {{GPUShaderModule}}
 
             Issue: Describe {{GPUDevice/createShaderModule()}} algorithm steps.
+
+            Note: User agents **should not** include error messages or shader text in
+            the {{GPUValidationError/message}} text of validation errors arising here:
+            these details are accessible via {{GPUShaderModule/compilationInfo()}}.
+            User agents **should** surface human-readable, formatted error details to
+            developers for easier debugging (for example as a warning in the browser developer
+            console, expandable to show full shader source).
```

Comment:
Not really a problem with this PR, but a behavior consideration nevertheless: It's a little unfortunate that while we can suppress console warnings for standard validation errors by checking for an `uncapturederror` handler there's no way to do the same for shader module errors, so this text implies they should always be echoed to the console.

Reply:
That's a good point. Perhaps we need something under "if scope is not undefined" like "if an error is captured by a scope, it should not be shown in the developer console". But then I don't know how a developer opts back into seeing human-readable error details. Is it by `console.log(shaderModule)`? `console.log(await shaderModule.compilationInfo())`? `console.log(gpuValidationErrorFromCreateShaderModule)`?

Reply:
And how do we even inform developers of how they opt back in?

OTOH, given shader errors should be mostly nonexistent in production applications, maybe it's fine if they're always shown.

Reply:
Yeah, I don't see it as a big problem. Just found the discrepancy worth noting.

Reply:
I wrote some more text. Is it too much detail?